### ISSUE TITLE
Running when manifest URL does not match resource URI in document

### ISSUE THREAD

**Issue by jeswr** [rubensworks/rdf-test-suite.js#77]
In the case of the RDF* specifications, the manifest documents are located at URLs like:

https://w3c.github.io/rdf-star/tests/trig/syntax/manifest.ttl

But in the document they are referred to using a fragment identifier

https://w3c.github.io/rdf-star/tests/trig/syntax#manifest

Which results in the error

```bash
> rdf-test-suite spec/parser.js https://w3c.github.io/rdf-star/tests/trig/syntax/manifest.ttl -i '{ "format": "trig-star" }' -c .rdf-test-suite-cache/

Error: Could not find a resource https://w3c.github.io/rdf-star/tests/trig/syntax/manifest.ttl in the document at https://w3c.github.io/rdf-star/tests/trig/syntax/manifest.ttl
    at ManifestLoader.<anonymous> (/home/jeswr/Documents/rdfstar/N3.js/node_modules/rdf-test-suite/lib/ManifestLoader.js:51:27)
    at Generator.next (<anonymous>)
    at fulfilled (/home/jeswr/Documents/rdfstar/N3.js/node_modules/rdf-test-suite/lib/ManifestLoader.js:5:58)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

Is there an option to override the resource URI (I realise that this is currently a problem with the way the spec is published but it would be good to have a way to get around it until then).

**Comment by rubensworks** [comment]
Many (most?) test suites unfortunately follow this convention.

There should be a workaround for that [here](https://github.com/rubensworks/rdf-test-suite.js/blob/master/lib/ManifestLoader.ts#L48-L49). Perhaps it needs to be adapted slightly to cope with this case.

**Comment by jeswr** [comment]
Linked issues but didn't follow up with comments.

The rdfstar test suite uses #manifest in the resource URI but /manifest.(jsonld|ttl) in the document URL.

There is a discussion around this (https://github.com/w3c/rdf-star/issues/269) and solution in (https://github.com/rubensworks/rdf-test-suite.js/pull/78).

**Comment by rubensworks** [comment]
This was fixed in #78.
